gnupg.c

```diff
@@ -885,6 +885,37 @@ PHP_FUNCTION(gnupg_keyinfo)
 			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, disabled, gpgme_subkey);
 			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, expired, gpgme_subkey);
 			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, revoked, gpgme_subkey);
+			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, can_certify, gpgme_subkey);
+#if GPGME_VERSION_NUMBER >= 0x000405  /* GPGME >= 0.4.5 */
+			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, can_authenticate, gpgme_subkey);
+#endif /* gpgme >= 0.4.5 */
+#if GPGME_VERSION_NUMBER >= 0x010100  /* GPGME >= 1.1.0 */
+			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, is_qualified, gpgme_subkey);
+#endif /* gpgme >= 1.1.0 */
+#if GPGME_VERSION_NUMBER >= 0x010900  /* GPGME >= 1.9.0 */
+			PHP_GNUPG_ARRAY_ADD_ASSOC_BOOL(subkey, is_de_vs, gpgme_subkey);
+#endif /* gpgme >= 1.9.0 */
+			/*
+				https://github.com/gpg/gpgme/blob/f7700a016926f0d8e9cb3c0337837deb7fe01079/src/gpgme.h.in#L258
+				https://github.com/gpg/gpgme/blob/f7700a016926f0d8e9cb3c0337837deb7fe01079/src/gpgme.c#L1196
+				printf '0x%02x%02x%02x\n' 1 2 0
+			*/
+			PHP_GNUPG_ARRAY_ADD_ASSOC_LONG(subkey, pubkey_algo, gpgme_subkey);
```

Comment:
I think it would be also good to expose PK constants so it can be compared:

https://www.gnupg.org/documentation/manuals/gpgme/Public-Key-Algorithms.html

Reply:
yup 👍 was thinking about it - just not sure on naming convention?

Reply:
Think it's the same name except GNUPG is used instead of GPGME and class constants use just the name. See https://github.com/lifeofguenter/php-gnupg/blob/ce5d13941f851fffa63a799156712a44c29cc606/gnupg.c#L459-L506